From d5737c020a5e175a4fd10118708c9ae8a4e074e2 Mon Sep 17 00:00:00 2001
From: alisterburt <alisterburt@gmail.com>
Date: Fri, 19 Jan 2024 23:55:24 +0000
Subject: [PATCH] move self to emeritus (#323)

I saw Talley's PR at #322 and thought I should probably do the same -
excited to see where the project goes, keep fighting the good fight and
I'll see you all around :-)
---
 docs/community/team.md | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/docs/community/team.md b/docs/community/team.md
index c9f4531c..0b7d316d 100644
--- a/docs/community/team.md
+++ b/docs/community/team.md
@@ -4,7 +4,6 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 
 ## Current Core Developers
 
-- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - [@alisterburt](https://github.com/alisterburt)
 - [Andy Sweet](https://github.com/napari/napari/commits?author=andy-sweet) - [@andy-sweet](https://github.com/andy-sweet)
 - [Draga Doncila Pop](https://github.com/napari/napari/commits?author=DragaDoncila) - [@DragaDoncila](https://github.com/DragaDoncila)*
 - [Genevieve Buckley](https://github.com/napari/napari/commits?author=GenevieveBuckley) - [@GenevieveBuckley](https://github.com/GenevieveBuckley)
@@ -27,6 +26,8 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 - [Justine Larsen](https://github.com/napari/napari/commits?author=justinelarsen) - [@justinelarsen](https://github.com/justinelarsen)
 - [Ziyang Liu](https://github.com/napari/napari/commits?author=potating-potato) - [@potating-potato](https://github.com/potating-potato)
 - [Ahmet Can Solak](https://github.com/napari/napari/commits?author=AhmetCanSolak) - [@AhmetCanSolak](https://github.com/AhmetCanSolak)
+- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - [@alisterburt](https://github.com/alisterburt)
+
 
 <sub>* on the napari [steering council](https://napari.org/community/governance.html#steering-council).</sub>
 
